### PR TITLE
fix(DB/Zone):improved Blades Edge

### PR DESCRIPTION
**Pull request by Knindzagxg**
## Changes Proposed:
- Missing SAI creature updates.
- Improvements to existing if possible.

## Tests Performed:
- Applied SQL without problems.
- Tested Silkwing (21373) cause they should despawn after evade / 5sec.
- Tested Fel Cannon (22451) cause their movement should be disabled during combat.

## How to Test:
- Apply SQL(s).
- Restart Server (You can .reload creature_template for each creature as well, but will take time)
- Go to zone (Check PR name) and check creatures around - You can also spawn (.npc add) each by command.

## Source for Changes:
- [Wowhead](https://wowhead.com) Database and or comments.
- [WotlkDB](https://wotlkdb.com/) Multi checking trough Private DB's.
- [WowWiki](https://wowwiki-archive.fandom.com/wiki/Portal:Main) Can be useful for extra informations.
- [WowPedia](https://wow.gamepedia.com/Wowpedia) Cross checking trough comments if possible.
- [Retail](https://worldofwarcraft.com/en-gb/) In-game checking if previous don't have any required informations.

## Target Branch:
- [x] Master

## Operating System:
- Linux (Debian 10)

## How to Test AzerothCore PR's
When a PR is ready to be tested, it will be marked as **[Waiting to be tested]**.
You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:
http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1616252675947625100.sql [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+(19948, 0, 0, 0, 6, 0, 100, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - On Just Died - Say Line 1'),
+(19948, 0, 1, 2, 4, 0, 35, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - On Aggro - Say Line 0'),
+(19948, 0, 2, 0, 61, 0, 100, 1, 0, 0, 0, 0, 11, 34932, 32, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - On Aggro - Cast \'34932\''),
+(19948, 0, 3, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 34802, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - Between 20-80% Health - Cast \'34802\' (No Repeat)'),
+(19948, 0, 4, 0, 2, 0, 100, 1, 5, 30, 0, 0, 11, 37786, 32, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Bloodmaul Skirmisher - Between 5-30% Health - Cast \'37786\' (No Repeat)');
```
Comment (pklloveyou) [inline data/sql/updates/pending_db_world/rev_1616252675947625100.sql:297]: It already has AI

Comment (pklloveyou) [reply data/sql/updates/pending_db_world/rev_1616252675947625100.sql:297]: I don't know if it's better than it is now

**Comment by pklloveyou** [comment]
First of all, it's better to have AI than none. I'll list individual creatures that have AI.

File: data/sql/updates/pending_db_world/rev_1616252675947625100.sql (hunk 1 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1616252675947625100.sql:297]: Yeah, those i updated had kinda bad AI.
Overlaping with spells mostly.

**Comment by Helias** [comment]
@azerothcore/testers if you can test please :pray:

**Review by Helias — APPROVED** [review]

**Comment by Sedatyf** [comment]
It is in progress on my side. I have to finish it but I can say that there is some bugs already. If you want, I can send you already what I found, or if you prefer, I can send it when it's finished

**Comment by Knindzagxg** [comment]
Send issues once you finish all mate xP

**Comment by Kitzunu** [comment]
update?

**Comment by FrancescoBorzi** [comment]
@Sedatyf

**Comment by Sedatyf** [comment]
There is a lot to check in this one. I have nearly finished it

**Comment by Sedatyf** [comment]
Hey guys sorry for the long delay. Here is my test about this PR:
Unbound Ethereal ✅
Bash'ir Spell-Thief ✅
Lightning Wasp ❓ it says it supposed to it on aggro but it spammed me with it. So should it be on combat? Because no repeat seems weird
Eredar Stormbringer ✅
Searing Elemental ❌ doesn't cast Fire Blast at all
Felstorm Overseer ✅
Insidious Familiar ✅ maybe put same id as the next one for shadow bold?
Felstorm Corruptor ✅
Trigul ✅
Darkflame Infernal ✅
Scorch Imp ✅
Doomforge Engineer ✅
Anguer guard ✅
Felsworn Scalewing ✅
Fel corrupter ❌ i think they shouldn't have the 36274 spell. Furthermore they don't cast out of combat
Felsworn Daggermaw ❌ i think you're using wrong charge spell id. Right now it's using warrrior spell id 35570 but it should use another one, like this one 34846. It's working, but it's weird 
Fiendling Flesh Beast ✅
Angered Arakkoa Protector ✅ they're level 11 but according to a WotLK DB, it should be like this
Vekh'nir Matriarch ✅
Vekh'nir Keeneye ✅
Vekh'nir Stormcaller ✅
Vekh'nir Dreadhawk ✅
Dullgrom Dredger ✅
Silkwing Larva ❓ the silkwing pops when larva is under 20%. But larva doesn't die, so we have two ennemies. Shouldn't the larva disappear if the butterfly appears?
Silwing ✅
Death's watch ✅
Death's Might ✅
Void Terror ✅
Rip-Blade Ravager ✅
Daggermaw Lashtail ❌ same as Felsworn Daggermaw
Ridgespine stalker ✅
Crystal Flayer ✅
Lesser Nether Drake ✅
Gore-Scythe Ravager ✅
Rema ✅
Bladewing Bloodletter ❌ it just spams Blood Leech (37838) like hell. I'm not sure this is normal behavior
Daggermaw Blackhide ✅
Outraged Raven's Wood Sapling ✅
Ruuan'ok Matriarch ✅
Ruuan'ok Ravenguard ✅
Ruuan'ok Cloudgazer ✅
Ruuan'ok Skyfury ✅
Fingrom ✅
Bladespire ravager ✅
Dorgok ✅
Bloodmaul lookout ✅
Bloodmaul skirmisher ❌ it doesn't say any sentence on aggro or when dying
Mugdorg ✅
Droggam ✅
Glumdor ❌ doesn't say anything
Gorr'dim ✅
Korgaah ✅
Bladespire Sober Defender ✅
Bladespire Raptor ✅
Young Crust Burster ✅
Wyrmcult Hunter ✅
Wyrmcult Hewer ✅
Grishna Matriarch ✅
Raven's Wood Stonebark ✅
Boulder'mok Chieftain ✅
Boulder'mok Shaman ✅
Boulder'mok Brute ✅
Dire Raven ✅
Wyrmcult Blessed ✅
Wyrmcult Provisioner ✅
Wyrmcult Acolyt ✅
Wyrmcult Zealot ❌ you made a typo on one of its spells. In database, it casts 37653 but this spell doesn't exist. It should casts 37635
Wyrmcult Scout ❌ has the right id but doesn't cast 37635
Fel Rager ✅
Legion Fel Cannon MKII ❌ there is this weird guy you can't kill as a regular player
image
Bladespine Basilisk ❌ it does cast the charge, but there is no animation at all. Which makes it weird in a way
Gan'arg Analyzer ❌ they have a weird spell (42415) and does not use Trash on when between 20-80% health
Nightmare Imp ❌ do not cast Shadow Bolt
Vile Fire-Soul ✅
Terrordar the Tormentor ✅
Lashh'an Kaliri ✅
Lashh'an talonite ✅
Lashh'an windwalker ✅
Lashh'an wing guard ✅
Mature Cavern Crawler ✅
Cavern Crawler ✅

**Comment by Knindzagxg** [comment]
Awesome mate xD
Ill check everything today once im back from work and update where need to.

**Comment by Knindzagxg** [comment]
Lightning Wasp - It's a shield that reflect everything for 10 minutes (Surrounds the caster with charged skin that has a 100% chance of striking melee or ranged attackers for 2 damage. Thus, the shield expires after 10 min.) Nothing to change.

Searing Elemental - Remove AI (Was added later after Wotlk)

Fell Corrupter - It's event when they walk near Raptors, and sometimes they will enrage them (Increases melee, ranged, and spell casting speed by 35% for 20 sec.) - Event is there since TBC but doesn't happen always. Nothing to change.

Felsworn Daggermaw - Whole point of Charge is to stun target xP Spell u linked will work only on target itself, which is bad xP Nothing to change.

Daggermaw Lashtail - Same as Daggermaw.

Bladewing Bloodletter - Sorted leech - Was missing "No Repeat" my bad xD

Bloodmaul Skirmisher - They won't say it always. It's chance from creature_text.

Glumdor - Same as Skirmisher, there is a chance for it, so we avoid spams.

Wyrmcult Zealot - Fixed typo error in ID xP

Wyrmcult Scout - Same as above, it's event trigger, so it won't happen always.

Legion Fel Cannon MKII - It's working for me. And in picture you linked PH NPC, which players can't even see (It's used for events or etc)

Bladespine Basilisk - Can't do anything about animation, it's core related.

Gan'arg Analyzer - Spell is used for event, when they check player faction. Removed Trash tho, it's added later on.

Nightmare Imp - Removed SAI -> Added in later expansions as well.

Everything is sorted now xP

**Comment by Knindzagxg** [comment]
Updated Silkwing Larva as well -> Once event trigger, Larva should be instantly dead as Silkwing appears.

**Comment by Sedatyf** [comment]
Perfect, I'll check those changes asap!

**Comment by Knindzagxg** [comment]
Need checking of those few changes that were requested, so i can move to Northrend Zones.

**Comment by Knindzagxg** [comment]
Retested required changes -> Everything working now as suggested.

With this one TBC should be fully updated - Rare Elites were updated last year, and time to move to Northrend zones.

**Pull request merged**